host/Manager.ts

```diff
@@ -215,10 +313,15 @@ export class Manager extends BaseExecutor {
         // but in case they are, we override them here.
         const sessionPermitted = { ...sessionRequested, ...user.session }
 
+        let { id, name } = sessionPermitted
+
         // Assign a unique, difficult to guess, identifier
         // that allows routing back to the `Session` instance
         // in the execute() method
-        const sessionId = crypto.randomBytes(32).toString('hex')
+        if (id === undefined) id = crypto.randomBytes(32).toString('hex')
+
+        // Assign a human friendly name if necessary
+        if (name === undefined) name = moniker.choose()
```

Comment:
Docker already generates these kinds of names, could this just grab the docker name if defined?

Reply:
Yeah, but also wanted to have the human friendly names for Firecracker sessions